changelog.txt

```diff
@@ -1,123 +1,314 @@
-0.4.1
-
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========
+
+Highlights
+----------
+	- Remove acceleration as supported property.
+	- Remove the read_gsd_shape_data flag from GSD reader.
+	- Raise AttributeError if accessing an invalid property.
+	- Add GETAR file reader/writer.
+	- Add support for various particle properties
+	- Extend reader/writer shape support
+	- Add automatic format detection to reader/writer
+	
+
+[0.5.0] -- 2019-0x-xx
+---------------------
+Added
++++++
+
+	- Add GETAR file reader/writer.
+	- Rowan is now a dependency.
+	- Add shape_dict representation to Shape classes.
+	- Add support for particle properties: 
+		- mass 
+		- charge 
+		- diameter
+		- image
+		- moment of inertia 
+		- angular momentum
+	- Add support for reading/writing shapes in GSD via hpmc state.
+	- Add universal reader/writer with format detection.
+	- Extend list of supported shape classes: 
+		- Ellipsoid 
+		- orientable spheres
+		- polygons
+		- spheropolygon
+		- convex polyhedron
+		- convex spheropolyhedron
+
+Changed
++++++++
+	- Improve unit test coverage.
+	- Revise documentation.
+	- Move shape definitions to separate module.
+	- Raise AttributeError if accessing an invalid property.
+	- Rename several existing shape classes.
+
+Deprecated
+++++++++++
+	- Tests for Python 2 are no longer updated (Python 2 support will be
+	entirely dropped in a future release).
+
+Removed
++++++++
+	- Remove acceleration as supported property.
+	- Remove the read_gsd_shape_data flag from GSD reader.
+
+Version 0.4
+===========
+
+Highlights
+----------
+	- GetarFileReader skips records that have a non-empty group field.
+	- Add CIF readers
+	- Add GSD writer
+	- Add shape support for shperopoly, polyunion, and convex polyhedron
+	union
+	- Add `gf2pos` script to convert any supported format to POS
```

Comment:
Minor quirk of reStructured Text: to get `monospace code font`, you wrap with TWO backticks instead of one (as in Markdown and GitHub comments like this one).

It should read ` ``gf2pos`` `.

Reply:
(Also search for this elsewhere in the file, and add double-backticks around any other class or variable names that you think should be in code font in the docs. See signac's changelog for examples.)

Reply:
I think I covered everything in a30acb5